giskard/llm/talk/utils/shap.py

```diff
@@ -0,0 +1,91 @@
+from typing import TYPE_CHECKING
+
+import numpy as np
+
+if TYPE_CHECKING:
+    from giskard import Dataset
+    from giskard.models.base import BaseModel
+
+from giskard.core.errors import GiskardImportError
+
+
+def _calculate_shap_values(data_to_explain: "Dataset", model: "BaseModel", dataset: "Dataset") -> np.ndarray:
```

Comment:
No need for " if you use TYPE_CHECKING

Reply:
I disagree. If I do not use ", then I get:
`NameError: name 'Dataset' is not defined` as an example.

Reply:
https://stackoverflow.com/questions/53421444/what-is-the-benefit-of-from-future-import-annotations-if-classes-are-impor

Reply:
Hmm, have to check on how we use it in Giskard, because we did not put the quotes, and not the future annotations either

Reply:
Will put back the `__future__ import annotations` to remove "